modular_skyrat/modules/organs/code/liver.dm

```diff
@@ -0,0 +1,7 @@
+obj/item/organ/liver/alien/roundstart
+	name = "adapted liver"
+	icon_state = "liver-x"
+	desc = "A liver that is adapted to take heavy intake of toxins in trade of being generally weaker and high chance to reject its owner if its filter too many toxins and being highly sensitive to EMP and ION."
```

Comment:
desc = "A liver that is adapted to handle heavy intake of toxins in exchange for being generally weaker, with a high chance to reject its owner if it filters too many toxins whilst being weak to EMP's."